### PR TITLE
Updates for ocean ice interpolation utility.

### PR DESCRIPTION
**Pull request by aerorahul**
This PR:
- updates the template for the namelist
- infers the vertical levels for `srcdims` from the input file when processing ocean data.  ice does not have levels.
- fixes a bug that allows interpolating from any model grid other than `mx025`.

### CODE REVIEW

**Review by WalterKolczynski-NOAA — APPROVED** [review]
LGTM

File: src/ocnicepost.fd/init_mod.F90 [hunk 1]
```diff
@@ -78,12 +78,11 @@ subroutine readnml
     close (iounit)
     nxt = srcdims(1); nyt = srcdims(2)
     nxr = dstdims(1); nyr = dstdims(2)
```
Comment (DeniseWorthen) [inline src/ocnicepost.fd/init_mod.F90:80]: Is nlevs (the number of vertical levels for the ocean model) no longer needed or used?

Comment (aerorahul) [reply src/ocnicepost.fd/init_mod.F90:80]: it is needed and used.  
It is just not being provided via the namelist.  It is being read in from the `ocean.nc` file in [ocnicepost.F90](https://github.com/NOAA-EMC/gfs-utils/pull/45/files#diff-c64691f1cb9756e5e066f35c8a4c3cd3610b87420d39df627ce4bc56474c3e40)

Comment (DeniseWorthen) [reply src/ocnicepost.fd/init_mod.F90:80]: I see. Thanks.

**Review by DeniseWorthen — APPROVED** [review]

**Pull request merged**